src/CMakeLists.txt

```diff
@@ -115,10 +115,6 @@ IF(BUILD_GUI)
    ENDIF(OPENGL_FOUND)
 ENDIF(BUILD_GUI)
 
-#Library: flann
-FIND_PACKAGE(PkgConfig)
```

Comment:
It seems that Flann uses headers only and libflann-dev installs the headers in /usr/include, so as long as libflann-dev is installed it should be fine without adding anything to CMakeLists.txt.